astropy/visualization/tests/test_time.py

```diff
@@ -1,20 +1,16 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-# ruff: noqa: E402
-
-import pytest
-
-pytest.importorskip("matplotlib")
-
 from contextlib import nullcontext
 
-import matplotlib.dates
-import matplotlib.pyplot as plt
+import pytest
 from erfa import ErfaWarning
 
 from astropy.time import Time
 from astropy.visualization.time import time_support
 
+matplotlib = pytest.importorskip("matplotlib")
+plt = matplotlib.pyplot
```

Comment:
If the `MPL_EPOCH_1970` flag is removed then there is no need to access `matplotlib.dates` anymore, which means these two lines can be combined into 
```suggestion
plt = pytest.importorskip("matplotlib.pyplot")
```

Reply:
That is outside the scope of the astropy/astropy#15646

Reply:
Removing `MPL_EPOCH_1970` is definitely outside the scope of astropy/astropy#15646, which is why I've opened a separate pull request for that. And although the difference between our suggestions is very small, the cost of rebasing a ~10 line pull request is also very small, so if we can shorten the code by an extra line without code churn then we should.